### PR TITLE
Smoke test Kotlin 1.8.0-RC

### PR DESCRIPTION
**Pull request by eskatos**
See https://github.com/JetBrains/kotlin/releases/tag/v1.8.0-RC

Requires changes to Santa Tracker

* https://github.com/gradle/santa-tracker-android/pull/23

### CODE REVIEW

**Comment by eskatos** [comment]
@bot-gradle test AST

**Review by alllex — APPROVED** [review]
LGTM!

**Comment by eskatos** [comment]
@bot-gradle test and merge

File: subprojects/docs/src/docs/userguide/compatibility.adoc [hunk 1]
```diff
@@ -46,7 +46,7 @@ For older Gradle versions, please see the table below which Java version is supp
 
 [[kotlin]]
 == Kotlin
-Gradle is tested with Kotlin 1.6.10 through 1.7.22.
+Gradle is tested with Kotlin 1.6.10 through 1.8.0. Beta and RC versions may or may not work.
```
Comment (ljacomet) [inline subprojects/docs/src/docs/userguide/compatibility.adoc:49]: 💄 One sentence per line

**Review by ljacomet — APPROVED** [review]
LGTM!

Aside from my favorite `.adoc` comment:

**Comment by eskatos** [comment]
@bot-gradle test AFT

**Comment by eskatos** [comment]
@bot-gradle test AFT

**Comment by eskatos** [comment]
@bot-gradle test AFT

**Comment by eskatos** [comment]
@bot-gradle test and merge

**Comment by eskatos** [comment]
@bot-gradle test and merge

**Comment by eskatos** [comment]
@bot-gradle test RFN

**Comment by eskatos** [comment]
@bot-gradle test APT

**Comment by eskatos** [comment]
@bot-gradle test AST

**Comment by eskatos** [comment]
@bot-gradle test APT

**Comment by eskatos** [comment]
@bot-gradle test this

**Comment by eskatos** [comment]
@bot-gradle APT

**Comment by eskatos** [comment]
@bot-gradle test APT

**Comment by eskatos** [comment]
@bot-gradle test RFN

**Review by ljacomet — APPROVED** [review]

**Comment by eskatos** [comment]
@bot-gradle test and merge

**Pull request merged**